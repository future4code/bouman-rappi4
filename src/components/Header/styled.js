import styled from "styled-components";

export const StyledImg = styled.img `
    position: fixed;
    top: 10px;
    left: 5px;
`

export const StyledHeader = styled.header `
    width: 100%;
    height: 40px;
    border-bottom: 1px solid lightgray;
    display: flex;
    align-items: center;
    justify-content: center;
    position: fixed;
    top: 0;
    background-color: #ffffff;
    z-index: 3;

`