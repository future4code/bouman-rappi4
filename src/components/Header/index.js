import React from 'react';
import { StyledHeader } from '../../style/styled';
import Back from '../../imagens/Ã­cones/back.png';
import { StyledImg } from './styled'

function Header(props) {
    return (
    <StyledHeader>
        <StyledImg src={props.img} onClick={props.onClick}/>
        <p>{props.title}</p>
    </StyledHeader>
    )
}

export default Header;
