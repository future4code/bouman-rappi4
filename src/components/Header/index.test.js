import React from 'react'
import renderer from "react-test-renderer";
import Header from './index';

describe ('teste componente Header', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Header></Header>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})