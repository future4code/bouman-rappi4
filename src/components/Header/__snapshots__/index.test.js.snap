// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente Header renders correctly 1`] = `
<header
  className="sc-bwzfXH RIwcK"
>
  <img
    className="sc-bdVaJa gSjNzR"
  />
  <p />
</header>
`;
